pkg/cable/vxlan/vxlan.go

```diff
@@ -558,5 +569,15 @@ func parseSubnets(subnets []string) []net.IPNet {
 func (v *vxlan) Cleanup() error {
 	klog.Infof("Uninstalling the vxlan cable driver")
 
-	return netlinkAPI.DeleteIfaceAndAssociatedRoutes(VxlanIface, TableID) // nolint:wrapcheck  // No need to wrap this error
+	err := netlinkAPI.DeleteIfaceAndAssociatedRoutes(VxlanIface, TableID)
+	if err != nil {
+		return errors.Wrapf(err, "unable to delete interface %s and associated routes from table %d", VxlanIface, TableID)
```

Comment:
Lets not return if there are any errors. Just log an Error message and we can continue with deleting the remaining items. General idea is to cleanup everything programmed by Submariner, but if there are any transient errors log the error and continue deleting whatever is possible.